USFMConverter/UI/Pages/ProjectDetailScreen.axaml.cs

```diff
@@ -139,14 +122,20 @@ private async void OnConvertStart(object? sender, RoutedEventArgs e)
 
         private void OnStartNewProject(object? sender, RoutedEventArgs e)
         {
-            ((Window)this.VisualRoot).DataContext = new ViewData();
+            // Only remove files, keep settings
+            var dataContext = (ViewData)DataContext;
+            dataContext.Files.Clear();
+
+            // reset assign data context to update UI automatically 
+            ((Window)this.VisualRoot).DataContext = null;
```

Comment:
yeah it's an awkward way of doing things like ReactJS: 
`this.setState(object)` instead of `this.state.object = object`. The latter won't update the UI since the state doesn't actually changes but only its property.